src/Congo.WebApi/Congo.WebApi.csproj

```diff
@@ -5,8 +5,14 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="MediatR.Extensions.Microsoft.DependencyInjection" Version="9.0.0" />
```

Comment:
We should also specify the base MediatR NuGet package as a dependency instead of relying on the implicit one from the DI package.

Reply:
There is no need for that, MediatR is included in this package just like with the Automapper package.
https://www.nuget.org/packages/MediatR.Extensions.Microsoft.DependencyInjection/

Reply:
It only specifies a dependency on the MediatR base package >= 9.0.0, so from my understanding we'd potentially upgrade unknowingly if Jimmy decides to release a new version.

Reply:
I agree with Thorsten, I'm not a big fan of implicit package references, at least not for important things like this.